src/pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -513,59 +546,71 @@ def __init__(self, extra_options):
                 )
 
         # Options not yet compatible with contact resistance
-        if options["contact resistance"] == "true":
-            if options["operating mode"] == "explicit power":
+        _ensure_tuple(options["contact resistance"])
+        if "true" in options["contact resistance"]:
+            _ensure_tuple(options["operating mode"])
+            if "explicit power" in options["operating mode"]:
```

Comment:
```suggestion
        _ensure_tuple(options["contact resistance"])
        _ensure_tuple(options["operating mode"])
        if "true" in options["contact resistance"]:
            if "explicit power" in options["operating mode"]:
```

Would it make sense to use your helper function outside of conditional statements, so that the option is ensured to be a tuple regardless of whether the condition is hit?

Reply:
Yes, I agree. In that case, the nested conditions could also consolidated in a single line